app/src/main/java/com/example/awillis/encorelink/FacilityAuthenticationActivity.java

```diff
@@ -0,0 +1,18 @@
+package com.example.awillis.encorelink;
```

Comment:
It might be a good idea to start adding top header comments with file name, purpose of the file or class, author's name and date so we know who to talk to when we scale to more files:
```

/**
FacilityAuthenticationActivity.java

Activity page to handle the login page for facility coordinators.
Created: 2/20/2018
Author:  baghaii
*/
```

Reply:
These headers are automatically added by my version of Android Studio. I am not trying to steal the authentication tasks from you.

Reply:
This is a general comment for all files that we author. It would make it easy to find out who to talk to if ever there are dependency questions or anything else.